src/components/views/rooms/JumpToBottomButton.js

```diff
@@ -18,11 +18,17 @@ import { _t } from '../../../languageHandler';
 import AccessibleButton from '../elements/AccessibleButton';
 
 export default (props) => {
+    let className = 'mx_JumpToBottomButton';
     let badge;
+
+    if (props.highlight) {
+        className += ' mx_JumpToBottomButton_highlight';
+    }
+
```

Comment:
this would be much cleaner using the `classNames` package which we use elsewhere in the app :)

Reply:
You got it!